storybook/src/components/DescriptionList/DescriptionList.stories.tsx

```diff
@@ -3,25 +3,27 @@
  * Copyright Gemeente Amsterdam
  */
 
+import { Link, Paragraph, UnorderedList } from '@amsterdam/design-system-react'
 import { DescriptionList } from '@amsterdam/design-system-react/src'
 import { Meta, StoryObj } from '@storybook/react'
 import { inverseColorDecorator } from '../shared/decorators'
-import { exampleParagraph } from '../shared/exampleContent'
-
-const paragraph = exampleParagraph()
 
 const meta = {
   title: 'Components/Text/Description List',
   component: DescriptionList,
   decorators: [inverseColorDecorator],
   args: {
     children: [
-      <DescriptionList.Term key={1}>Gebied</DescriptionList.Term>,
-      <DescriptionList.Details key={2}>Gemeente Amsterdam</DescriptionList.Details>,
-      <DescriptionList.Term key={3}>Stadsdeel</DescriptionList.Term>,
-      <DescriptionList.Details key={4}>West</DescriptionList.Details>,
-      <DescriptionList.Term key={5}>Opmerkingen</DescriptionList.Term>,
-      <DescriptionList.Details key={6}>{paragraph}</DescriptionList.Details>,
+      <DescriptionList.Term key={1}>het hoger onderwijs</DescriptionList.Term>,
```

Comment:
Why did you change this example? I kind of prefer the old one

Reply:
It was a bit contrived. ‘Gebied’ for Gemeente Amsterdam? Mentioning 4 city districts while we have 8, some with incorrect names.

I wanted a good example of the generic meaning of a description list. This content is straight from our writing guide and matches that goal: it *describes terms* that we preferred often-used ones.